packagelist

```diff
@@ -1,277 +1,278 @@
-fennel
-fennel-bin
-fishui-git
-fm6000-git
-fnlfmt
-forticlient-deb
-freetube-deb
-fsearch-git
-gcmcore-deb
-geekbench-bin
-git
-git-delta-deb
-git-extras-git
-gitblade-deb
-gitfiend-deb
-github-cli-deb
-github-desktop-deb
-gitkraken-deb
-glow-deb
-gnome-shell-extension-installer-git
-gnome-tweaks
-go-bin
-gomuks-deb
-google-chrome-beta-deb
-google-chrome-deb
-google-chrome-unstable-deb
-google-earth-pro-stable-deb
-graillon
-grapejuice-git
-grive2
-grive2-git
-gtkhash
-heroic-games-launcher-deb
-htop
-httptoolkit-deb
-hyper-deb
-hyperfine-bin
-i3-gaps
-i3lock-color
-imwheel-exclude-patched-git
-insync-deb
-ivpn-deb
-ivpn-ui-deb
-jetbrains-toolbox-bin
-keepassxc
-kvantum
-kxstudio-repos-deb
-lagrange
-latte-dock-git
-lbry-deb
-lemonbar-xft-git
-libcutefish-git
-libndi4-deb
-libresprite-git
-libressl
-librewolf-app
-lrz-syncshare-deb
-lsd-deb
-lutris-deb
-lyrebird-git
-lyx
-maitreya8-deb
-makedeb-deb
-mangohud
-micro
-microsoft-edge-beta-deb
-microsoft-edge-dev-deb
-microsoft-edge-stable-deb
-microsoft-prod-repo
-minecraft-launcher
-minecraft-launcher-deb
-minecraft-pi-reborn-app
-moka-icon-theme
-ms-teams-deb
-mullvad-vpn-beta-deb
-mullvad-vpn-deb
-mutt-wizard
-nala-deb
-nano
-neofetch
-neovim
-neovim-git
-nodejs-lts-deb
-nofetch
-noisetorch-bin
-nomachine-deb
-notion-git
-nowpm
-nyxt-deb
-obs-backgroundremoval
-obs-backgroundremoval-git
-obs-ndi-deb
-oil
-onefetch-bin
-onnxruntime-1-7-0-bin
-opendoas-git
-opensnitch-deb
-opensnitch-ui-deb
-opentoonz
-opentoonz-gts
-opera-beta-deb
-opera-deb
-opera-developer-deb
-pacget-git
-paconvert-git
-pacstall-gui-git
-panagement
-paper-icon-theme
-papirus-icon-theme
-pdf-mix-tool-git
-pdfmm-git
-pfetch-bin
-pipes.sh-bin
-pipewire
-planner-git
-plexmediaserver-deb
-polybar
-pomatez-deb
-popsicle
-proton-python-client
-protonmail-bridge-deb
-protonvpn-cli
-protonvpn-gui
-protonvpn-nm-lib
-pycharm-community-bin
-qogir-gtk-theme-bin
-qogir-icon-theme-bin
-qtile-git
-renegate
-reshade-shaders-git
-sed
-shell-color-scripts
-shellcheck-bin
-shellharden-bin
-shfmt-bin
-slack-deb
-slop-git
-smartgit
-smplayer-deb
-soundux
-spacecadetpinball-git
-spotify-client-deb
-st-distrotube-git
-st-lukesmith-git
-steam-deb
-steam-launcher-deb
-steamtinkerlaunch-git
-stellarium
-surf
-surf-distrotube-git
-suru-icon-theme-git
-taskwarrior
-teams-for-linux-deb
-telegram-bin
-tenacity-git
-thrive-deb
-tidal-hifi-bin
-timeshift
-tmpmail-bin
-tmux
-topgrade-bin
-tor
-torbrowser-launcher-deb
-torguard-deb
-touchegg-deb
-treefetch-bin
-tuner
-tuner-git
-tutanota-app
-ulauncher-deb
-universal-ctags-git
-uwufetch
-vega-strike-engine-git
-vega-strike-git
-veracrypt-deb
-vim
-viper4linux
-virtualbox-deb
-vivaldi-deb
-vivarium
-vscode-deb
-vscodium-deb
-waf-git
-walc-app
-warpinator-deb
-wazzapp-deb
-webapp-manager
-webapp-manager-deb
-windscribe-gui-deb
-windscribe-legacy-cli-deb
-wine-staging-git
-yabridge
-yad
-yafetch-git
-yandex-browser-beta-deb
-yandex-browser-stable-deb
-youtube-dl-git
-zentile-bin
-zotero
+ adapta-gtk-theme-git
```

Comment:
What??

Reply:
This is what happened when I ran `ls -1 packages > packagelist`, as mentioned on the page [Pacscript 101](https://github.com/pacstall/pacstall/wiki/Pacscript-101), in the "updating the packagelist" section. If you count the lines there's only 1 line added.

Reply:
Oh wait, I didn't see it added a symbol in the beginning of every line, sorry, I'll fix it.